stl/src/stacktrace.cpp

```diff
@@ -85,49 +86,61 @@ namespace {
             debug_symbols = nullptr;
         }
 
+        if (dbgeng != nullptr) {
+            FreeLibrary(dbgeng);

```

Comment:
[`FreeLibrary()`](https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-freelibrary) returns a `BOOL` indicating success; ignoring it in `uninitialize()` seems reasonable, but we should probably say `(void)` to explicitly indicate that we're doing so (and to avoid code analysis warnings in the future).